selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/inject/SeleniumTestHandler.java

```diff
@@ -335,4 +343,9 @@ private void shutdown() {
 
   /** Returns parent injector. */
   public abstract Injector createParentInjector();
+
+  /** Returns list of child modules */
+  public List<Module> getChildModules() {
```

Comment:
Rename and make it abstract
`getTestClassModules`

Reply:
This method is aimed to provide just child modules, and the list is only used as an argument of method **injector.createChildInjector()**. So, could you, please, explain, why we should rename it to **getTestClassModules** which can includes any kind of modules including parent ones?

Also, if we make **getTestClassModules** an abstract, we obligate implementations to realize this method to return empty list in case of Codenvy Onprem and Codenvy SaaS tests.

Reply:
`getChildModules` ok

Reply:
okay